app/View/CoPetitions/json/resend.ctp

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * COmanage Registry CoPetition JSON resend View
+ *
+ * Portions licensed to the University Corporation for Advanced Internet
+ * Development, Inc. ("UCAID") under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * UCAID licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @link          http://www.internet2.edu/comanage COmanage Project
+ * @package       registry
+ * @since         COmanage Registry v0.1
+ * @license       Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ */
+
+// Get a pointer to our model
+$model = $this->name;                     // Unlike in the controllers, this is CamelPlural
+$req = Inflector::singularize($model);
+$modelpl = Inflector::tableize($req);
+$modelcc = $model;
+
+$response = array();
```

Comment:
maybe this can be done
$response = (!empty($vv_response) ? $vv_response : array())
?

Reply:
Done